README.md

```diff
@@ -61,41 +174,33 @@ command-line executable (compatible with the one in linux).
     If you want to integrate the compiler into a larger Java program, depend on
     `//:compiler_shaded_deploy.jar` instead.
 
-### Using an IDE
-
-See [Bazel IDE Integrations](https://docs.bazel.build/versions/master/ide.html)
+### Using a terminal
 
-## Running
+You can trigger the build process easily with package.json scripts or by calling
+Bazel manually. 
 
-On the command line, at the root of this project, type
+```bash
+# bazel build //:compiler_shaded_deploy.jar
+yarn build
 
-```sh
-bazel-bin/compiler_unshaded_deploy.jar
+# bazel build :all
+yarn build:all
 ```
 
-This starts the compiler in interactive mode. Type
-
-```javascript
-var x = 17 + 25;
-```
+### Using an IDE
 
-then hit "Enter", then hit "Ctrl-Z" (on Windows) or "Ctrl-D" (on Mac or Linux)
-and "Enter" again. The Compiler will respond:
+See [Bazel IDE Integrations](https://docs.bazel.build/versions/master/ide.html).
 
-```javascript
-var x=42;
-```
+## Running
 
-The Closure Compiler has many options for reading input from a file, writing
-output to a file, checking your code, and running optimizations. To learn more,
-type
+Once the compiler has been built, the compiled JAR will be in the `bazel-bin/`
+directory. You can access it with a call to `java -jar ...` or by using the
+package.json script:
 
+```bash
+# java -jar bazel-bin/compiler_unshaded_deploy.jar [...args]
```

Comment:
Am I mistaken that the deploy JAR is self-executable? Something about ELF...

Reply:
IIRC it *purported* to be executable, but did not seem to be:
```none
christian@workspaces:~/PersonalProjects/closure-compiler$ ./bazel-bin/compiler_unshaded.jar
./bazel-bin/compiler_unshaded.jar: line 1: $'PK\003\004': command not found
./bazel-bin/compiler_unshaded.jar: line 2: META-INF/��PK: No such file or directory
./bazel-bin/compiler_unshaded.jar: line 3: zD�PPMETA-INF/MANIFEST.MFManifest-Version:: No such file or directory
./bazel-bin/compiler_unshaded.jar: line 4: Created-By:: command not found
./bazel-bin/compiler_unshaded.jar: line 5: Target-Label:: command not found
./bazel-bin/compiler_unshaded.jar: line 6: $'\r': command not found
./bazel-bin/compiler_unshaded.jar: line 7: $'PK\001\002': command not found
./bazel-bin/compiler_unshaded.jar: line 9: META-INF/��PK: No such file or directory
./bazel-bin/compiler_unshaded.jar: line 11: zD�PP+META-INF/MANIFEST.MFPK}�: No such file or directory
```